Sources/JXKit/JXContext.swift

```diff
@@ -4,24 +4,63 @@ import JavaScriptCore
 #else
 import CJSCore
 #endif
-#if canImport(FoundationNetworking)
-import FoundationNetworking
+#if canImport(Foundation)
+import Foundation
 #endif
 
 /// A JavaScript execution context. This is a cross-platform analogue to the Objective-C `JavaScriptCore.JSContext`.
 ///
 /// This wraps a `JSGlobalContextRef`, and is the equivalent of `JavaScriptCore.JSContext`
 open class JXContext {
+    /// Context configuration.
+    public struct Configuration {
+        /// Whether scripts evaluated by this context should be assessed in `strict` mode. Defaults to `true`.
+        public var strict: Bool
+        
+        /// Whether hot reloading of JavaScript script resources is enabled.
+        public var isHotReloadEnabled: Bool {
+#if canImport(Foundation)
```

Comment:
All the script loading uses URLs and so is conditional on Foundation support, which was a pain. I did test by turning all these into '#if false' to make sure it all still compiles and tests and runs.

Reply:
I think we can dispense with `#if canImport(Foundation)` altogether and just `import Foundation`. While it is theoretically possible that we might run without Foundation, it exists for all the platforms we support, and so we might as well just import it.

Reply:
Or, perhaps, just `import struct Foundation.URL` at the top to point out that we only have a minor dependency on the framework.